package.json

```diff
@@ -5,8 +5,9 @@
     "test": "npx lerna exec npm t",
     "build": "npx lerna run build",
     "docs": "npx typedoc",
-    "publish:release-candidate": "npx lerna publish --allow-branch release-candidate --dist-tag next --preid rc --no-verify-access --conventional-prerelease",
-    "publish:latest": "npx lerna publish --allow-branch main --create-release github --no-verify-access --conventional-graduate",
+    "publish:release-candidate": "npx lerna publish --allow-branch release-candidate --dist-tag next --preid rc --no-verify-access --conventional-prerelease bump preminor",
+    "publish:release": "npx lerna publish --allow-branch main --dist-tag latest --no-verify-access --conventional-graduate bump minor",
+    "publish:hotfix": "npx lerna publish --allow-branch $(git rev-parse --abbrev-ref HEAD) --no-verify-access --conventional-graduate bump patch",
```

Comment:
should we just drop the `--allow-branch` flag entirely if we're always taking the current branch anyway?

Reply:
Yes 👍🏼